dotnet_sdk/spec/spec_helper.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: strict
```

Comment:
Running spoom in CI insisted that the file `dotnet_sdk/spec/spec_helper.rb` be bumped from `typed: true` to `typed: strict`. This was blocking.